src/domain/collaboration/canvas/CanvasesManagement/CanvasesManagementViewWrapper.tsx

```diff
@@ -40,13 +39,14 @@ const CanvasesManagementViewWrapper: FC<CanvasesManagementViewWrapperProps> = ({
   ...canvasesState
 }) => {
   const { isFeatureEnabled } = useConfig();
-  if (!calloutId || loadingCanvases) {
-    return <Loading />;
+  if (!calloutId) {
+    return null;
```

Comment:
Maybe a TODO show something while the callout/canvas information loads but for now this is okay, the dialog will not be shown until everything is ready, and there is already a loading animation on the container of CanvasesManagementViewWrapper